external/intern_video2_multimodal.patch

```diff
@@ -0,0 +1,389 @@
+diff --git a/InternVideo2/multi_modality/__init__.py b/InternVideo2/multi_modality/__init__.py
+index 8b13789..dcbf4ed 100644
+--- a/InternVideo2/multi_modality/__init__.py
++++ b/InternVideo2/multi_modality/__init__.py
+@@ -1 +1,22 @@
+ 
++# Import main models
++from .models.internvideo2_stage2_visual import InternVideo2_Stage2_visual
++from .models.internvideo2_stage2_audiovisual import InternVideo2_Stage2_audiovisual
++from .models.internvideo2_clip import InternVideo2_CLIP
++from .models.internvideo2_clip_small import InternVideo2_CLIP_small
++
++# Import backbone components
++from .models.backbones.internvideo2 import pretrain_internvideo2_1b_patch14_224, pretrain_internvideo2_6b_patch14_224
++from .models.backbones.bert.builder import build_bert
++from .models.backbones.internvideo2.pos_embed import interpolate_pos_embed_internvideo2_new
++
++__all__ = [
++    'InternVideo2_Stage2_visual',
++    'InternVideo2_Stage2_audiovisual', 
++    'InternVideo2_CLIP',
++    'InternVideo2_CLIP_small',
++    'interpolate_pos_embed_internvideo2_new',
++    'pretrain_internvideo2_1b_patch14_224',
++    'pretrain_internvideo2_6b_patch14_224',
++    'build_bert'
++]
+\ No newline at end of file
+diff --git a/InternVideo2/multi_modality/configs/internvideo2_mm_config_model.json b/InternVideo2/multi_modality/configs/internvideo2_mm_config_model.json
+new file mode 100644
+index 0000000..40d0640
+--- /dev/null
++++ b/InternVideo2/multi_modality/configs/internvideo2_mm_config_model.json
```

Comment:
nit: For all the files that we've added new and didn't exist in the original repo is it possible to also add the boilerplate Nvidia license header with the patch?